build.sbt

```diff
@@ -5,6 +5,7 @@ lazy val root = project
   .settings(
     name := "rock-paper-scissor",
     scalaVersion := "2.12.3",
+    resolvers += "Artima Maven Repository" at "http://repo.artima.com/releases",
```

Comment:
this shouldn't be needed

Reply:
both aritma supersafe and scalactic where "highly recommended" by `scalatest` so I plugged them in (although I still don't use them..)

Reply:
Nah, I disagree with Bill :P